src/runtime-rs/Makefile

```diff
@@ -147,6 +147,7 @@ DEFMAXMEMSZ := 0
 ##VAR DEFBRIDGES=<number> Default number of bridges
 DEFBRIDGES := 0
 DEFENABLEANNOTATIONS := [\"kernel_params\"]
+DEFENABLEANNOTATIONS_COCO := [\"kernel_params\",\"cc_init_data\"]
```

Comment:
I think we need to add `cc_init_data` into https://github.com/kata-containers/kata-containers/blob/c92bb1aa88fa75005524575de3d192fe747dd3e7/src/runtime/Makefile#L220
as well (and I would support considering renaming `DEFENABLEANNOTATIONSTEE` to  `DEFENABLEANNOTATIONS_COCO` as well)